pkg/operation/botanist/component/machinecontrollermanager/machine_controller_manager.go

```diff
@@ -271,6 +295,15 @@ func (m *machineControllerManager) emptyDeployment() *appsv1.Deployment {
 	return &appsv1.Deployment{ObjectMeta: metav1.ObjectMeta{Name: v1beta1constants.DeploymentNameMachineControllerManager, Namespace: m.namespace}}
 }
 
+func (m *machineControllerManager) emptyPodDisruptionBudget() client.Object {
+	objectMeta := metav1.ObjectMeta{Name: v1beta1constants.DeploymentNameMachineControllerManager, Namespace: m.namespace}
+
+	if m.runtimeVersionGreaterEqual123 {
```

Comment:
policy/v1 is available from Kubernetes version 1.21 onwards.
```suggestion
	if m.runtimeVersionGreaterEqual121 {
```